tests/ui/get-20-records.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from "playwright/test";
+
+let apiContext;
+
+test.beforeAll(async ({ playwright }) => {
+  apiContext = await playwright.request.newContext({
+    baseURL: "https://127.0.0.1:5000/",
+    extraHTTPHeaders: {
+      Accept: "application/vnd.inveniordm.v1+json",
+      Authorization: `token ${process.env.API_TOKEN}`,
+    },
+  });
+});
+
+test.afterAll(async ({}) => {
+  await apiContext.dispose();
+});
+
+test("redirection to search page with 20 records", async ({ page }) => {
+  await page.goto("/");
+
+  await page
+    .locator('form[role="search"]:not(.sidebar form) .ui.button')
+    .click();
+
+  await expect(page.getByTestId("result-item")).toHaveCount(10);
+
+  const resPerPage = page.locator('.computer div[role="listbox"]');
+
+  await resPerPage.click();
+
+  await resPerPage.locator("span", { hasText: "50" }).click();
```

Comment:
I want 50 records but as a next step I expect to get 20??

Reply:
This is 50 records per page. This scenario is on purpose for being sure that there are only 20 records, because we don't have count component in restoration project. And if during the test we select 20 records per page then it could be possible that some records are just not shown